packages/bpk-component-map/readme.md

```diff
@@ -0,0 +1,81 @@
+# bpk-component-map
+
+> Backpack map component.
+
+## Installation
+
+```sh
+npm install bpk-component-map --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkMap from 'bpk-component-map';
+
+export default () => (
+  <BpkMap
+    containerElement={<div style={{ height: '400px' }} />}
+    mapElement={<div style={{ height: `100%` }} />}
+    defaultZoom={15}
+    defaultCenter={{
+      lat: 27.9881,
+      lng: 86.925,
+    }}
+    options={{
+      zoomControl: false,
+      dragEnabled: false,
+    }}
+  />
+);
+```
+
+## Accompanying HOCs
+
+### withScriptjs
+
+`withScriptjs` is a HOC that loads the Google Maps Javascript, then loads the map. This is useful for when you don't already have the Google Maps Javascript loaded.
+
+If you intend to include multiple maps on one page, it's better to load the Google Maps Javascript elsewhere and not use this HOC, as it downloads the script every time it's used.
+
+```js
+import React from 'react';
+import BpkMap, { withScriptjs } from 'bpk-component-map';
+
+const MAP_URL = 'https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places';
+
+const BpkMapWithScript = withScriptjs(BpkMap);
+
+export default () => (
+  <BpkMapWithScript
+    googleMapURL={MAP_URL}
+    loadingElement={<div />}
+    containerElement={<div style={{ height: '400px' }} />}
+    mapElement={<div style={{ height: `100%` }} />}
+    defaultZoom={15}
+    defaultCenter={{
+      lat: 27.9881,
+      lng: 86.925,
+    }}
+    options={{
+      zoomControl: false,
+      dragEnabled: false,
+    }}
+  />
+);
+```
+
+## Props
+
+### BpkMap
+
+| Property	      | PropType	| Required                	| Default Value |
```

Comment:
apart from this misalignment 

![bitmoji](https://render.bitstrips.com/v2/cpanel/ea7ab00a-8055-4f6b-9933-d47005a9b8d4-7fa1897d-dae4-4909-a4ec-c165763ceef8-v1.png?transparent=1&palette=1&width=246)

Reply:
Weird! It's not misaligned in my editor:

![screen shot 2018-05-24 at 15 50 02](https://user-images.githubusercontent.com/73652/40493232-2909714e-5f6a-11e8-83d1-8f95cb39670f.png)